components/SearchForm.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<div class="search-container">
-		<form id="search-form" action="">
+		<form id="search-form" action="/search">
```

Comment:
is `/search` a method that will have the proper search functionality?

Reply:
The search path, as defined in pages, is `search`: https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/master/pages/search/_query/index.vue